documentation/markup.rst

```diff
@@ -10,6 +10,29 @@ This document describes the custom reStructuredText markup introduced by Sphinx
 to support Python documentation and how it should be used.
 
 
+Quick Reference
+===============
+
+This table summarizes which markup should be used for some commonly used
+elements:
+
+================ ========================================= ====================
+Element          Markup                                    See also
+================ ========================================= ====================
+arguments        ``*arg*``                                 :ref:`inline-markup`
```

Comment:
```suggestion
parameters       ``*arg*``                                 :ref:`inline-markup`
```

Shouldn't this be parameters *of* the function, not arguments *to* the function?

Reply:
I guess it could be both, because if the function defines a *foo* parameter, it also accepts a *foo* argument.  Using "parameters" might lead people to think that they should use the markup in the function signature.  Perhaps we could specify both as "arguments/parameters"?

Reply:
> I guess it could be both, because if the function defines a foo parameter, it also accepts a foo argument. 

Yeah, I guess so—unless it is positional-only, where the name only matters for the parameter.

> Using "parameters" might lead people to think that they should use the markup in the function signature.

Well, it seems rather unlikely at least to me that a Python documented would adopt this misconception, particularly if they looked at any existing function signature in the docs, but its not impossible I suppose.

> Perhaps we could specify both as "arguments/parameters"?

That sounds good, we'd just need to widen the table column—which would also give more room for my combined suggestion.